wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -79,6 +79,12 @@
         {% endblocktrans %}
     </noscript>
 
+    <div data-ie11-warning hidden>
+        <div class="capabilitymessage">
+            {% blocktrans with url="https://docs.wagtail.io/en/stable/editor_manual/browser_issues.html#ie11" %}Wagtail is gradually removing support for IE11, which may affect you or your organisation. <a href="{{ url }}">Learn more about our IE11 support plans</a>.{% endblocktrans %}
+        </div>
+    </div>
```

Comment:
Since it's fairly common (and tolerated) for developers to rebrand all references to Wagtail in the UI, we should probably put this in a `{% block %}` tag. (Admittedly that does make it somewhat trivial for them to hide the message too, but then they'd be shooting themselves in the foot...)